openreview/venue_request/venue_request.py

```diff
@@ -710,6 +711,46 @@ def setup_venue_deployment(self):
                 }
             ))
 
+    def setup_venue_post_submission(self):
+
+        post_submission_content = {
+            'force': {
+                'value-radio': ['Yes', 'No'],
+                'required': True,
+                'description': 'Force to create the blind submission if conference if double-blind'
+            },
+            'hide_fields': {
+                'values-regex': '.*',
```

Comment:
This needs to be `value-regex`, otherwise the value for `hide_fields` is saved as a list instead of a string. This causes failure down the line when the process function of `post_submission` invitation calls `split` over the value of this field (and `split` can not be called on a `list`).

Reply:
Thanks for caching this. I removed the split call.